src/components/templates/AddWalletToList/index.tsx

```diff
@@ -52,6 +52,8 @@ export const AddWalletToList = (props: AddWalletToListProps): JSX.Element => {
                 fontFamily="Inter_500Medium"
                 fontWeight="500"
                 color={COLORS.smokyBlack}
+                numberOfLines={1}
+                style={{ width: '50%' }}
```

Comment:
Instead of `width: "50%"`, we can have flex: 1 and horizontal spacing (with value scale(24)) between `list.name` and `list.totalBalance`.